.github/workflows/release.yml

```
@@ -5,8 +5,8 @@ on:
     branches-ignore:
       - '**'
     tags:
-      - (0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)
-      - (0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)-(alpha|rc)[1-9]\d*
+      - '[0-9]+.[0-9]+.[0-9]+'
+      - '[0-9]+.[0-9]+.[0-9]+-(alpha|rc)[0-9]+'
```

Ah, it's glob-ish something. I don't think `()` gonna work here. How about this:

```suggestion
      - '[0-9]+.[0-9]+.[0-9]+'
      - '[0-9]+.[0-9]+.[0-9]+-{alpha,rc}[0-9]+'
```